app/db.py

```diff
@@ -0,0 +1,81 @@
+import sqlite3
+import random
+from app.product.models import ProductModel
+
+def getNewId():
+    return random.getrandbits(28)
+
+def connect():
+    conn = sqlite3.connect('products.db')
+    cur = conn.cursor()
+    cur.execute("""CREATE TABLE IF NOT EXISTS products 
+                (id INTEGER PRIMARY KEY,
+                product_name TEXT,
+                category TEXT,
+                producer TEXT,
+                description TEXT,
+                price FLOAT)""")
```

Comment:
Jeśli nie korzystasz z ORM to można wyciągnąć queries do osobnego folderu i plików